From ca5880dabd565128655e52738ca6824fb9e91ed1 Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Mon, 20 Mar 2023 07:56:30 +0200
Subject: [PATCH] [SW-136144] gaudi3: include register header files

The header files included in gaudi3_reg_map files
in embedded specs requires them to copy to ami build
package directory for hl-fw-loader. Therefore in-order
to remove it from gaudi3 specs adding the required
file compilation

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: Ie56a6fdd8df39a03eb0ecd5335bb4496b06fc193
---

diff --git a/arch/gaudi3/arc/onix.c b/arch/gaudi3/arc/onix.c
index 9dcc0bb..bbb4fc3 100644
--- a/arch/gaudi3/arc/onix.c
+++ b/arch/gaudi3/arc/onix.c
@@ -3,6 +3,7 @@
  * (C) Copyright 2022 HabanaLabs, Ltd.
  */
 
+#include <gaudi3_blocks_linux_driver.h>
 #include <global_conf_regs.h>
 #include <arc_regs.h>
 #include <gaudi3_reg_map.h>
